recipes/vulkan-memory-allocator/all/conanfile.py

```diff
@@ -29,7 +29,7 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("vulkan-headers/1.3.243.0")
+        self.requires("vulkan-headers/[>=1.0]")
```

Comment:
This is more likely to create conflicts in Conan graph resolution than avoid them, unfortunately. Can't recommend.

https://github.com/conan-io/conan-center-index/issues/26512#issuecomment-2631405093

Reply:
@amitiunin - your understanding is correct.